src/translations/enTranslations.json

```diff
@@ -486,6 +486,14 @@
         "personalia": "Personalia",
         "roles": "Roles"
       },
+      "identity": {
+        "full_name": "Full name",
+        "hide_full_nin": "Hide national identity number",
```

Comment:
hide/show full nin har vi allerede fra før, så kanskje gjenbruk den, eller flytt den til `common`

Reply:
samme med `national_identity_number`. Ser ikke for meg at dette er noe som skal ha ulike tekster i fremtiden, så da kan vi gjenbruke i stedet 👍